firmware/user/modes/mode_tunernome.c

```diff
@@ -340,8 +374,44 @@ static void ICACHE_FLASH_ATTR tunernomeUpdate(void* arg __attribute__((unused)))
         default:
         case TN_TUNER:
         {
+            clearDisplay();
+
+            // Instructions at top of display
+            plotText(0, 0, "Blu= Flat   Wht= OK   Red= Sharp", TOM_THUMB, WHITE);
```

Comment:
Alternative idea, use the same 'needle' as the metronome, where centered is tuned, right is sharp, and left is flat?

Reply:
I mean in addition to the LEDs, not replacing them

Reply:
This would involve finding some more screen real estate, or reorganizing what's there now. I had planned to add the 6 guitar notes on the sides of the word "GUITAR" arranged in the same shape as the LEDs, so people know which LED is which string. I also wanted to potentially add a little sensitivity printout on screen somewhere, which is already getting cramped.

Reply:
Fair, it was just a thought